hedera-node/hedera-addressbook-service-impl/src/test/java/com/hedera/node/app/service/addressbook/impl/test/handlers/NodeGeInfoHandlerTest.java

```diff
@@ -0,0 +1,240 @@
+        givenValidNode(true);
+        final var responseHeader = ResponseHeader.newBuilder()
+                .nodeTransactionPrecheckCode(ResponseCodeEnum.OK)
+                .build();
+        final var expectedInfo = getExpectedInfo(true);
+
+        readableNodeState = readableNodeState();
+        given(readableStates.<EntityNumber, Node>get(NODES_KEY)).willReturn(readableNodeState);
+        readableStore = new ReadableNodeStoreImpl(readableStates);
+
+        final var config =
+                HederaTestConfigBuilder.create().withValue("ledger.id", "0x03").getOrCreateConfig();
+        given(context.configuration()).willReturn(config);
+
+        final var query = createGeNodeInfoQuery(nodeId.number());
+        when(context.query()).thenReturn(query);
+        when(context.createStore(ReadableNodeStore.class)).thenReturn(readableStore);
+
+        final var response = subject.findResponse(context, responseHeader);
+        final var nodeInfoResponse = response.nodeGetInfoOrThrow();
+        assertEquals(ResponseCodeEnum.OK, nodeInfoResponse.header().nodeTransactionPrecheckCode());
+        assertEquals(expectedInfo, nodeInfoResponse.nodeInfo());
+    }
+
+    @Test
+    @DisplayName("OK response is correctly handled in findResponse")
+    void getsResponseIfOkResponse() {
+        givenValidNode();
+        final var responseHeader = ResponseHeader.newBuilder()
+                .nodeTransactionPrecheckCode(ResponseCodeEnum.OK)
+                .build();
+        final var expectedInfo = getExpectedInfo(false);
+
+        final var query = createGeNodeInfoQuery(nodeId.number());
+        when(context.query()).thenReturn(query);
+        when(context.createStore(ReadableNodeStore.class)).thenReturn(readableStore);
+
+        final var config =
+                HederaTestConfigBuilder.create().withValue("ledger.id", "0x03").getOrCreateConfig();
+        given(context.configuration()).willReturn(config);
+
+        final var response = subject.findResponse(context, responseHeader);
+        final var nodeInfoResponse = response.nodeGetInfoOrThrow();
+        assertEquals(ResponseCodeEnum.OK, nodeInfoResponse.header().nodeTransactionPrecheckCode());
+        assertEquals(expectedInfo, nodeInfoResponse.nodeInfo());
+    }
+
+    private NodeInfo getExpectedInfo(boolean deleted) {
+        return NodeInfo.newBuilder()
+                .nodeId(nodeId.number())
+                .accountId(accountId)
+                .description("description")
+                .gossipEndpoint((List<ServiceEndpoint>) null)
+                .serviceEndpoint((List<ServiceEndpoint>) null)
+                .gossipCaCertificate(Bytes.wrap(gossipCaCertificate))
+                .grpcCertificateHash(Bytes.wrap(grpcCertificateHash))
+                .weight(0)
+                .ledgerId(new BytesConverter().convert("0x03"))
+                .deleted(deleted)
+                .build();
+    }
+
+    private Query createGeNodeInfoQuery(final long nodeId) {
```

Comment:
Typo in method name, you're missing a 't'

Reply:
fixed thanks